spec/factories/collection_types.rb

```diff
@@ -1,7 +1,8 @@
 FactoryGirl.define do
   factory :collection_type, class: Hyrax::CollectionType do
-    sequence(:id) { |n| n }
-    sequence(:title) { |n| "Title #{n}" }
+    sequence(:id) { |n| format("%010d", n + 2) }
```

Comment:
Factory Girl starts the sequence at 1.  If the Admin Set and User Collection get created via engine cart generation process, there will already be collection types and the new collection type won't save because you can't change those.  I'm not sure what happens when the db is cleared between tests to thing generated before the tests run.